hpi-schul-cloud/nuxt-client
# BC-6496-Unify-language-keys-for-Whiteboard

# Short Description

QA note:
Tested change one env:
https://bc-6496-unify-language-keys.nbc.dbildungscloud.de/
No issues found. See jira: https://ticketsystem.dbildungscloud.de/browse/BC-6496





## Links to Ticket and related Pull-Requests


- https://ticketsystem.dbildungscloud.de/browse/BC-6496


## Changes



## Data-security



## Deployment



## New Repos, NPM packages or vendor scripts



## Screenshots of UI changes



## Checklist before merging


> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed
